choose_latest_release.bash

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-echo "== Choosing latest release (7.x-branch)"
+echo "== Choosing latest release (13.x-branch)"
```

Comment:
this should be remove all together.
IMO choosing the LLVM commit should be done outside of those scripts since bolt is upstreamed